cli/internal/codegen/testdata/expected_typescript.ts

```diff
@@ -91,11 +16,20 @@ export namespace svc {
     }
 
     export interface Request {
-        Foo: Foo
+        Foo: Foo | undefined
```

Comment:
Would we prefer expressing this as `Foo?: Foo` instead? (are they 100% equivalent?)

Reply:
For the record; they aren't 100% equivalent, but we discussed the behaviour differences on a call and decided to revert this change